filebeat/module/elasticsearch/audit/_meta/fields.yml

```diff
@@ -6,15 +6,15 @@
       description: "The layer from which this event originated: rest, transport or ip_filter"
       example: "rest"
       type: keyword
-    - name: origin_type
+    - name: origin.type
       description: "Where the request originated: rest (request originated from a REST API request), transport (request was received on the transport channel), local_node (the local node issued the request)"
       example: "local_node"
       type: keyword
-    - name: realm
+    - name: user.realm
       description: "The authentication realm"
       example": "active_directory"
       type: keyword
-    - name: roles
+    - name: user.roles
```

Comment:
Is the idea with these two two be ecs compatible? If yes, these are not on the root level but nested under elasticsearch.audit.

Reply:
No, these are meant to be under `elasticsearch.audit`. I looked at https://github.com/elastic/ecs and didn't find a `user.roles` in there.

Reply:
Yeah this looks like they are in the right place, under es.audit.*